app/services/dttp/register_for_trn.rb

```diff
@@ -22,10 +22,13 @@ def call
       batch_response = batch_request.submit
 
       entity_ids = OdataParser.entity_ids(batch_response: batch_response)
-      # TODO: make this a state transition method
-      trainee.update!(dttp_id: entity_ids["contacts"],
-                      placement_assignment_dttp_id: entity_ids["dfe_placementassignments"],
-                      submitted_for_trn_at: Time.zone.now)
+
+      trainee.transition_to!(:submitted_for_trn) && trainee.update!(
```

Comment:
Same as above - should the transition and update be part of a sinle action on a trainee?